packages/0x.js/src/index.ts

```diff
@@ -68,7 +68,7 @@ export {
     MetamaskSubprovider,
 } from '@0x/subproviders';
 
-export { AbiDecoder } from '@0x/utils';
+export { AbiDecoder, TransactionData } from '@0x/utils';
```

Comment:
@hysz kinda tangential to your PR but mind also exporting `transactionHashUtils` from `0x.js`? We forgot to do so.